docs/infra/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: Quickstart | zkSync Docs
+---
+
+# Quick Start Guide for zkSync Node
+
+## Prerequisites
+
+- **Installations Required:**
+  - [Docker Compose](https://docs.docker.com/compose/install/)
+  - [Docker](https://docs.docker.com/get-docker/)
+
+## Running a zkSync Node Locally
+
+### Starting the Node
+
+- **For a Mainnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file mainnet-external-node-docker-compose.yml up
+  ```
+
+- **For a Testnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file testnet-external-node-docker-compose.yml up
+  ```
+
+### Resetting the Node State
+
+- **For a Mainnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file mainnet-external-node-docker-compose.yml down --volumes
+  ```
+
+- **For a Testnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file testnet-external-node-docker-compose.yml down --volumes
+  ```
+
+- **To reset its state, run:**
```

Comment:
I believe you can delete this section, it's a copy/paste of `For a Testnet instance` above